src/plugins/sockets/inGameEvents.js

```diff
@@ -65,6 +65,7 @@ export async function handleInGameEvents(evData) {
       gameStore.processFives(evData.discardedCards, evData.game);
       break;
     case SocketEvent.RESOLVE:
+    case SocketEvent.FIZZLE:
```

Comment:
Need a socket event for fizzle in order for the client to respond to it. Or else we could have updated publish-game to use `SocketEvent.RESOLVE`, but I think this is better. The awkward bit is that it checks whether the one-off happened inside the switch case, instead of treating it as a separate case. We can clean this up upon switching to the gamestate api